docs/modules/block.md

```diff
@@ -893,6 +905,46 @@ data modify storage bs:in block.emit_block_particle merge value { delta: "0 0 0"
 function #bs.block:emit_block_particle
 ```
 
+::::
+::::{tab-item} Block Sound
+
+```{function} #bs.block:play_block_sound
+
+Play a block sound of the given block.
+
+:Inputs:
+  **Execution `at <entity>` or `positioned <x> <y> <z>`**: Position where the sound will be played.
+
+  **Storage `bs:in block.play_block_sound`**:
+  :::{treeview}
+  - {nbt}`compound` Block sound data
+    - {nbt}`string` **sound**: The sound to play. Usually took from the `sounds` property of the virtual block (cf [Get](#get) functions).
```

Comment:
The get reference causes a fatal sphinx error. It may be related to the treeview plugin but i guess the easiest solution is to remove the reference for now